### DISCUSSION TITLE
Include the possibility of associating users with contacts

### DISCUSSION THREAD

**Discussion by gabrielat** [netbox-community/netbox#17727] [Ideas]
My team and I would find great use in associating users to contacts.
We have been using custom fields to do so, but it has been a challenge..

Maybe other users would find it useful too, so here I am asking :-)

The idea would be like shown in the anexed image.
![example](https://github.com/user-attachments/assets/99a684e0-a9a2-41c1-ac97-7aa2bb6ebda1)

**Comment by jsenecal** [comment]
Would it make sense to make the contact a user's profile as well in that case ?

**Reply by gabrielat** [reply]
Yes, I think it would! Maybe even automatically create contacts based on users when added..
Just ventilating ideas 😂

**Reply by jscmenezes** [reply]
> Would it make sense to make the contact a user's profile as well in that case ?

Yes! It would be great!
